AMaDda/Screens/Main/PopupView/ProgressWithTextView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  ProgressWithTextView.swift
+//  AMaDda
+//
+//  Created by LeeSungHo on 2022/07/29.
+//
+
+import UIKit
+
+class ProgressWithTextView: UIView {
+
+    let contactGoalCount = UserDefaults.standard.contactGoalCount == 0 ? 1 : UserDefaults.standard.contactGoalCount
+    var userContactGoal = UserDefaults.standard.userContactGoal
+    private lazy var previousContactGoalCount = contactGoalCount - 1
+    private lazy var previousProgress = Float(previousContactGoalCount) / Float(userContactGoal)
+    private lazy var progress = Float(contactGoalCount) / Float(userContactGoal)
+    private let width = 220
+    
+    var ConsArray: [NSLayoutConstraint] = []
+    
+    // MARK: - property
+    private lazy var feedBackprogressView: UIProgressView = {
+        let progressView = UIProgressView()
+        progressView.progressViewStyle = .default
+        progressView.progress = previousProgress
+        return progressView
+    }()
+    private let progressImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = ImageLiterals.FeedBackLow1
+        imageView.tintColor = .black
+        return imageView
+    }()
+    private lazy var contactGoalCountLabel: UILabel = {
+        let label = UILabel()
+        label.text = "\(contactGoalCount-1)회"
+        return label
+    }()
+    private lazy var  userContactGoalLabel: UILabel = {
+        let label = UILabel()
+        label.text = "\(userContactGoal)회"
+        return label
+    }()
+    
+    // MARK: - init
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        print("progress \(userContactGoal)")
+        setUpNotification()
+        configureAddSubViews()
+        configureLayoutConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - selector
+    @objc private func showPopUpAnimation() {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
+            UIView.animate(withDuration: 1, delay: 0, options: .curveLinear) {
+                NSLayoutConstraint.deactivate(self.ConsArray)
+                self.ConsArray.popLast()
```

Comment:
p2;
Xcode 오류가 떠서..!
```suggestion
                _ = self.ConsArray.popLast()
```
이렇게 하면 오류가 지워집니다.